kfp/pipeline_generator/superpipeline/README.md

```diff
@@ -0,0 +1,3 @@
+## Steps to generate a new pipeline
```

Comment:
Maybe mention it support only kfp v1. Also, you can add a link to data-prep-lab/kfp/doc/multi_transform_pipeline.md for more info on super pipeline,,,

Reply:
Fixed.